features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
+    En de persoon heeft een ouder '2' met de volgende gegevens
+    | voornamen (02.10)  | geslachtsaanduiding (04.10) |
+    | John               | M                           |
+    En de ouder '2' is gecorrigeerd naar de volgende gegevens
+    | voornamen (02.10)  | geslachtsaanduiding (04.10) |
+    | Johanna            | V                           |
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                                                       |
     | type                | RaadpleegMetBurgerservicenummer                              |
-    | burgerservicenummer | 555550001                                                    |
+    | burgerservicenummer | 000007892                                                    |
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen,ouders.geslacht |
-    Dan heeft de ouder met ouderAanduiding '1' de volgende gegevens
+    Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
     | naam                  | waarde |
     | ouderAanduiding       | 1      |
     | geslacht.code         | V      |
     | geslacht.omschrijving | vrouw  |
-    En heeft de ouder met ouderAanduiding '1' volgende 'naam' gegevens
-    | naam      | waarde |
-    | voornamen | Noa    |
-    En heeft de ouder met ouderAanduiding '2' de volgende gegevens
-    | naam                  | waarde |
-    | ouderAanduiding       | 2      |
-    | geslacht.code         | V      |
-    | geslacht.omschrijving | vrouw  |
-    En heeft de ouder met ouderAanduiding '2' de volgende 'naam' gegevens
-    | naam      | waarde  |
-    | voornamen | Johanna |
+    | naam.voornamen        | Noa    |
+    En heeft de persoon een 'ouder' met de volgende gegevens
+    | naam                  | waarde  |
+    | ouderAanduiding       | 2       |
+    | geslacht.code         | V       |
+    | geslacht.omschrijving | vrouw   |
+    | naam.voornamen        | Johanna |
 
   @gba
   Scenario: adoptie
-    Gegeven de persoon met burgerservicenummer 999996629 heeft de volgende ouders in de registratie
-    | Categorie | Voornamen (02.10)  | Datum ingang familierechtelijke betrekking (62.10) | Onjuist (84.10) |
-    | 2         | Olivia Ghislaine   | 20190317                                           |                 |
-    | 52        | Me'Lisa            | 20170106                                           |                 |
-    | 52        |                    | 00000000                                           | O               |
-    | 3         | Kevin              | 20190317                                           |                 |
-    | 53        |                    |                                                    |                 |
-    | 53        |                    | 00000000                                           | O               |
+    Gegeven de persoon met burgerservicenummer '000007893' heeft een ouder '1' met de volgende gegevens
```

Comment:
de data in dit testgeval is onnodig ingewikkeld voor wat hier getoond moet worden. komt omdat het overgenomen was uit een bestaand geval in GBA-V proefomgeving.
Hier gebeuren namelijk twee dingen (3 stappen):
1. eerst opnemen onbekende ouder en dat vervolgens aanvullen met echte gegevens
2. daarna adoptie

De beginsituatie met volledig onbekende ouder is denk ik niet nodig. Is ook niet meer nodig, omdat we de testdata zelf opbouwen met een andere bsn.
Je kan beginnen met  ouder Me'Lisa, en dat daarna laten wijzigen (a.g.v. adoptie) in ouder Olivia Ghislaine.

Reply:
Ga ik in een nieuwe PR aanpassen.